src/components/app/routes/loaders/tests/rootLoader.test.jsx

```diff
@@ -351,7 +383,7 @@ describe('rootLoader', () => {
         if (!(isLinked || isStaffUser)) {
           expectedQueryCount = 2;
         }
-      } else if (hasResolvedBFFQuery) {
+      } else if (isMatchedRoute) {
```

Comment:
nit: `hasResolvedBFFQuery` is a bit more explicit of a name than `isMatchedRoute`

Reply:
went with `isMatchedBFFRoute`